sandpack-react/src/components/CodeEditor/CodeMirror.tsx

```diff
@@ -197,11 +197,27 @@ export const CodeMirror = React.forwardRef<CodeMirrorRef, CodeMirrorProps>(
         const customCommandsKeymap: KeyBinding[] = [
           {
             key: "Tab",
-            run: insertTab,
+            run: (view): boolean => {
+              indentMore(view);
```

Comment:
As `indentMore` returns true, this was preventing running the next command. So, here I'm grabbing the custom keybinding and running before returning it.

I'm not fully confident, but it seems to work